app/services/book_service.rb

```diff
@@ -28,11 +28,7 @@ def self.books_on_shelf(shelf, auth_token)
   end
 
   def self.add_book(shelf_id, volume_id, auth_token)
-    # shelf =  BookService.book_shelves(auth_token)[:items].find do |shelf|
-    #             shelf[:title] == "To read"
-    #           end
-    # shelf_id = shelf[:id]
-    conn(auth_token).get "/books/v1/mylibrary/bookshelves/#{shelf_id}/addVolume?volumeId=#{volume_id}"
```

Comment:
Poor shelves service, you were a massive waste of time...

Reply:
But they did such a good job.